src/main/java/org/kohsuke/github/GHWorkflow.java

```diff
@@ -122,8 +122,14 @@ public void dispatch(String ref) throws IOException {
      *            default properties configured in the workflow file will be used when inputs are omitted.
      * @throws IOException
      *             the io exception
+     * @throws IllegalArgumentException
+     *             the illegal argument exception
      */
-    public void dispatch(String ref, Map<String, Object> inputs) throws IOException {
+    public void dispatch(String ref, Map<String, Object> inputs) throws IOException, IllegalArgumentException {
```

Comment:
Adding an exception is technically an API change.  Also, `IllegalArgumentException` is a `RuntimeException` so it doesn't have to be part of the signature. 

```suggestion
    public void dispatch(String ref, Map<String, Object> inputs) throws IOException {
```